### ISSUE TITLE
Finding accuracy of the extracted answer span

### ISSUE THREAD

**Issue by linubajy** [rohankumar0002/extractAnswer-wikipedia#3]
Is there any way to find the accuracy or any other score metrics for the extracted answer in the model?

**Comment by rohankumar0002** [comment]
yes. Model returns a lot of logits. You can leverage that.
